backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -317,6 +324,7 @@ public AddTestResultResponse addMultiplexResult(
       order.setTestEventRef(savedEvent);
       savedOrder = _testOrderRepo.save(order);
       _testEventReportingService.report(savedEvent);
+      _fhirQueueReportingService.report(testEvent);
```

Comment:
An unfortunate side effect of some renamings during the initial multiplex migration is that `addMultiplexTest` is a total misnomer 😅 This is actually the code that runs when _any_ test is submitted through the queue, not just multiplex tests. Since we don't want to put covid-only results on the FHIR queue, you'll need to update this logic to include a disease check of some kind (more or less "if `results` include flu then report to FHIR").

(Side note that a more accurate method name would be something like `addTestResultPotentiallyForMultipleDiseases` but that's a mouthful)